OpenLIFULogin/OpenLIFULogin.py

```diff
@@ -650,6 +650,9 @@ def onSceneEndClose(self, caller, event) -> None:
             self.initializeParameterNode()
 
     def onDatabaseChanged(self, db: Optional["openlifu.db.Database"] = None):
+        if self._cur_login_state == LoginState.LOGGED_IN:
+            slicer.util.infoDisplay(f"Due to the database location being changed, you were logged out of {get_current_user().id}.")
```

Comment:
```suggestion
            slicer.util.infoDisplay(f"You have been logged out because the database location was changed.")
```

(just cleaning up the text a bit -- I don't know if it one is logged out "of" a user id. one is logged out "of" a system.)